.github/workflows/continuous-integration-e2e.yaml

```diff
@@ -6,78 +6,78 @@ env:
 
 on:
   pull_request:
-    branches: ['master']
+    branches: [ "master" ]
```

Comment:
This commit should probably go to a separate PR that sets up some tool/linter that enforces consistent yaml format, as well as formats all yaml files in the repository.

Reply:
It's already been fixed, I pushed one last commit. But I can create a PR with a rule.

Reply:
Create ticket: LW-6786